src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -542,8 +542,9 @@ public function column_imported( $post ) {
 		$now = current_time( 'timestamp', true );
 
 		$retry_time = false;
-		if ( $last_import_error ) {
-		    $retry_time = $record->get_retry_time();
+
+		if ( ! empty( $last_import_error ) ) {
```

Comment:
should it be changed on 526 as well?

Reply:
on line 526 we just set it, line 524, so it will be defined